provider/azure/storage.go

```diff
@@ -31,6 +31,7 @@ const (
 	//
 	// See: https://azure.microsoft.com/en-gb/documentation/articles/virtual-machines-disks-vhds/
 	volumeSizeMaxGiB = 1023
+	nvmeLunOffset    = 2
```

Comment:
Can we move this next to the other storage consts and add a comment. There's already a comment against the related nvmeDiskName const